modules/runtime/nuxeo-runtime-test/src/test/java/org/nuxeo/runtime/MockEventsInfo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.runtime;
+
+/**
+ * @since 11.5
+ */
+public class MockEventsInfo {
+
+    public int beforeStop = 0;
```

Comment:
Old code but `= 0` is redundant. You should probably keep Bogdan's name in the contributors as well.